setup.py

```diff
@@ -3,14 +3,19 @@
 
 from setuptools import setup
 
+# TODO: add meaningful long description
+# First, write it to Readme.md
+# then, copy it here and document in process to transfer all Readme updates
+long_description = "Software for test measurement equipment platform."
+
 # options reference: https://docs.python.org/2/distutils/
 setup(
     name="easy_phi",
-    packages=["easy_phi", 'static', 'templates', 'scripts'],
```

Comment:
@mansuleman movement of static into easy_phi folder without deletion from packages caused issue you mentioned in email. This is the fix I mentioned in response